api/app/src/main/kotlin/packit/service/UserService.kt

```diff
@@ -28,6 +31,7 @@ interface UserService
     fun checkAndUpdateLastLoggedIn(username: String)
     fun getServiceUser(): User
     fun getUserPrincipal(user: User): UserPrincipal
+    fun getAllNonServiceUsers(): List<User>
```

Comment:
i have actually made this in my refactor PR.. would be great if you can replace the code please with below!!! 

UserService
```
override fun getAllNonServiceUsers(): List<User>
    {
        return userRepository.findByUserSourceNotOrderByUsername("service")
    }
```
UserRepository
```
fun findByUserSourceNotOrderByUsername(userSource: String): List<User>
```